Sources/Jetworking/Globals/HTTPStatusCodeType.swift

```diff
@@ -0,0 +1,30 @@
+enum HTTPStatusCodeType {
+    case informational
+    case successful
+    case redirection
+    case clientError
+    case serverError
+    case unknown
+
+    init(statusCode: Int) {
```

Comment:
We should make it possible to access the statusCode in the Result as well! Sometimes it is necessary to differentiate between `Ok` and `Created` for example.

Reply:
I will add a ticket for this.

Reply:
Ok perfect!